tests/tests/kernel/datatypes/ezimage/ezimagealiashandler_regression.php

```diff
@@ -12,12 +12,6 @@ class eZImageAliasHandlerRegression extends ezpDatabaseTestCase
 {
     protected $backupGlobals = false;
 
-    public function __construct()
-    {
-        parent::__construct();
-        $this->setName( "eZImageAliasHandler Regression Tests" );
-    }
-
```

Comment:
You just need to pass along the parent constructor arguments, Yannick.

No, the name isn't useful, but it's easy to fix :-)